odata-csdl/15 Identifier and Path Values.md

```diff
@@ -28,6 +28,10 @@ restrictions:
 Non-normatively speaking it starts with a letter or underscore, followed
 by at most 127 letters, underscores or digits.
 
+For maximum interoperability services SHOULD use simple identifiers
+consisting only of latin letters, the underscore, and decimal numbers,
```

Comment:
```suggestion
consisting only of letters from the ASCII charset, the underscore, and decimal digits,
```
The Latin alphabet did not have the letters J, U, W.

Reply:
We use Unicode terminology here, see https://www.compart.com/en/unicode/U+004A, https://www.compart.com/en/unicode/U+0055, https://www.compart.com/en/unicode/U+0057